browser-fetching/index.md

```diff
@@ -71,6 +71,10 @@ const fetchDog = () => fetch('https://dog.ceo/api/breeds/image/random')
   <img id="dog" style="width: 100%;" alt="Random dog image" src="https://via.placeholder.com/400" />
 </div>
 
+The HTTP request to fetch the dog data is reflected in the _Network_ tab, and the data from the fetched response can be inspected in the Console panel. 
+Below is an example of the data in the Web Developer tools.
```

Comment:
I wasn't sure on this one, but bringing it to your attention.

`Web Developer tools` is what is used on this line. But on line 33, `_Web Developer Tools_` is used -- notice difference in caps on 'Tools'.

Reply:
`Displaying dynamic data that changes over time For example, fetching current weather conditions for the given location`

Up to you, but I feel like maybe we want more a more obvious visual demarcation where it starts `For example`. The italicizing seems to fit logically, but maybe something stronger visually, either instead of or in addition to the italicizing.

Reply:
A couple more suggestions:

1. Add a "Continue" button at the bottom of the page. doesn't ahve to go anywhere, but could send user to the analyte education module if you want.
2. I think it couldn't hurt to include a codepen after the lesson, and say something like "if you want to experiment more, you can do it either in the dev tools or go to this codepen" - the external link: https://codepen.io/ingrey1-the-typescripter/pen/vYRKepE.
  
     If you decide you like the codepen link idea, let me know if you want any changes to the JS, CSS, and HTML on the codepen instance.

Reply:
Thanks for creating the codepen--it looks great. It encourages experimentation, which really helps reinforce learnings. I'll link to it as suggested.